vastloader.go

```diff
@@ -0,0 +1,59 @@
+package vast
+
+import (
+	"context"
+	"io"
+	"net/http"
+	"time"
+)
+
+type VASTResponse struct {
+	Body []byte
+	Err  error
+}
+
+// VASTResponseHandler handles the repsonse from the request
+func VASTResponseHandler(res *http.Response, err error) VASTResponse {
+	if err != nil {
+		return VASTResponse{
+			Body: []byte(""),
+			Err:  err,
+		}
+	}
+	defer res.Body.Close()
+	body, err := io.ReadAll(res.Body)
+	if err != nil {
+		return VASTResponse{
+			Body: []byte(""),
+			Err:  err,
+		}
+	}
+	return VASTResponse{
+		Body: body,
+		Err:  nil,
+	}
+}
+
+// LoadVAST attempts to load VAST
+func LoadVAST(VASTAdTagURI string, milliseconds time.Duration) (*VAST, error) {
```

Comment:
```suggestion
func LoadURL(VASTAdTagURI string, milliseconds time.Duration) (*VAST, error) {
```
Same here, we can remove `VAST`. `Load` by itself is a bit generic though. Maybe `LoadURL` or `LoadAdTagURI`?